workspaces/vscode-python/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN find /etc/sudoers.d -type f -not -name README -delete && \
 USER root
 ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
+    apt-get install -y man-db unminimize \
```

Comment:
This is the same change that we had to make in #10728. Seems like something definitely changed with the Ubuntu base image.